xml/admin_install_salt.xml

```diff
@@ -726,8 +726,9 @@ done</screen>
      <filename>/srv/pillar/ceph/stack/ceph/cluster.yml</filename> manually and
      set the <option>cluster_network:</option> and
      <option>public_network:</option> options. Note that the file will not be
-     overwritten after you install <package>deepsea</package>.
+     overwritten after you install <package>deepsea</package>. Then, run:
     </para>
+<screen>chown -R salt:salt /srv/pill/ceph/stack</screen>
```

Comment:
I think that should be pillar, not pill, i.e. `/srv/pillar/ceph/stack/`. @swiftgist shouldn't the `/srv/pillar/ceph` directory also be salt-owned?

Reply:
That's what I get for copy/paste and no checking. Thanks @ddiss

Reply:
yeah, did I type that in the original bug? :man_facepalming: